packages/teleport/src/Audit/__snapshots__/Audit.story.test.tsx.snap

```diff
@@ -1009,7 +1157,7 @@ exports[`list of all events 1`] = `
         <td
           style="word-break: break-word;"
         >
-          User [alice] has send RPC Request [undefined] in database [master] on [sqlserver02]
+          User [alice] has sent RPC Request [undefined] in database [master] on [sqlserver02]
```

Comment:
could you check why this has an `undefined` value? its either our JSON fixture is incorrect, or we are accessing an incorrect field name

Reply:
Fixed the templated was outdated after switching from `porcname` to `proc_name`